### ISSUE TITLE
docs not loading fontawesome

### ISSUE THREAD

**Issue by nicolas-arnold** [sandstorm-io/sandstorm#3286]
This is a minor issue, but should also be an relatively easy fix.

The FontAwesome Font is not loaded. Checked on Firefox and Safari. Maybe something is off with the path, the version, or the file is just missing?

downloadable font: download failed (font-family: "FontAwesome" style:normal weight:400 stretch:100 src index:1): status=2147746065 source: https://docs.sandstorm.io/en/latest/fonts/fontawesome-webfont.woff2?v=4.7.0

**Comment by ocdtrekkie** [comment]
Yeah, it's known. I think it falls back to a woff(1) according to dev console.
